runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -167,6 +174,52 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 
 		switch (eventName)
 		{
+			case "geTagsActive":
+				// tell the script if it should use callbacks when searching and iterating items
+				String s = client.getVar(VarClientStr.INPUT_TEXT).toLowerCase();
+				intStack[intStackSize - 1] = s.startsWith("tag:") ? 1 : 0;
+				break;
+			case "searchGeTagItem":
+				String geSearch = stringStack[stringStackSize - 1];
+				geResults.clear();
+				geIdx = 0;
+
+				String tag = geSearch.replace("tag:", "");
+				List<Integer> tags = tagManager.getItemsForTag(tag);
+				boolean inMembersWorld = client.getWorldType().contains(WorldType.MEMBERS);
+
+				// filter out non-tradable items and members items when in a F2P world
+				tags = tags.stream()
+					.map(Math::abs)
```

Comment:
You should instead do reverse lookup in ItemVariationMapping here

Reply:
added item variation support